todo_app/app.py

```diff
@@ -1,7 +1,7 @@
 from flask import Flask, render_template, redirect, request
 
 from todo_app.flask_config import Config
-from todo_app.data.session_items import get_items, add_item
+from todo_app.data.trello_items import add_item, get_items, move_item_to_done
```

Comment:
Nice, the fact this file has such a small change is a good sign you designed your functions well 👍